### DISCUSSION TITLE
Linear connections | siblings

### DISCUSSION THREAD

**Discussion by hackmajoris** [markmap/markmap#141] [Ideas]
This is what I want to achieve:

<img width="691" alt="Screenshot 2022-11-18 at 13 41 22" src="https://user-images.githubusercontent.com/3758212/202697714-b96c3199-9f41-4ab4-9810-643357b874c9.png">

1. Linear lines.
2. Siblings connection between nodes.

I'll look into the source code to figure it out what can be done but if you have sugesstions, let's discuss.

**Comment by devx** [comment]
This feature would complete most of my use cases with mind map.

**Comment by IGitYou** [comment]
I am not a developer but would imagine this could be achieved by using section liking to headers .. You have two way arrows and so any linked section could draw a two way arch between the linked sections... right? Am I over simplifying this?
